ultraplot/colors.py

```diff
@@ -3156,7 +3157,7 @@ def _translate_key(self, original_key, mirror=True):
 
             # Try mirroring the non-lowered key
             if reverse:
-                original_key = original_key.strip("_r")
+                original_key = original_key.rstrip("_r")
```

Comment:
This method removes any of the characters in `_r`, not `_r` if the string ends with `_r`. I think you may want something like `original_key.rsplit("_r", maxsplit=1)[0]`

Reply:
I can also include it in here, although I think this issue will not be a major things as the `_r` is added by ultraplot. If the colormap ends with `_r` it will be listed as `{cmapnameendingwith_r}_r` which should then split fine I think.